presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaMetadata.java

```diff
@@ -303,8 +303,7 @@ public Set<QualifiedTablePrefix> calculatePrefixesWithTableName(
 
         return prefixes.stream()
                 .flatMap(prefix -> Stream.concat(
-                        metadata.listTables(session, prefix).stream(),
-                        metadata.listViews(session, prefix).stream()))
```

Comment:
I don't think its a redundant call. There are a few connectors where I can see listTables would not include the views as well. For eg. Accumulo, BigQuery.